Tools/WebKitTestRunner/InjectedBundle/cocoa/AccessibilityCommonCocoa.mm

```diff
@@ -66,10 +66,10 @@ Class webAccessibilityObjectWrapperClass()
 JSObjectRef makeJSArray(JSContextRef context, NSArray *array)
 {
     NSUInteger count = array.count;
-    JSValueRef arguments[count];
+    Vector<JSValueRef> arguments(count);
```

Comment:
JSValueRef is GC managed, so it needs to be stored on stack. So using heap Vector will cause GC crash.